### ISSUE TITLE
resource.modelName is dasherized improperly

### ISSUE THREAD

**Issue by psbanka** [salsify/ember-cli-pact#26]
Hello, sorry if this is a user-error question, but I cannot seem to figure out why it is that I have camelized model-names, but when doing pact-testing, I am getting errors because pact/mirage is searching for dasherized names.

My test looks like this:
```js
module('Pact | certificates', function(hooks) {
  setupTest(hooks)
  setupPact(hooks, { provider: 'my-api', consumer: 'my-app' })

  test('tls/certificates endpoint', async function(assert) {
    given('the customer exists')
    given('a generic certificate exists')
    const query = {filter: {'customer.id': '123456'}}
    const tlsCertificates = await interaction(() => this.store().query('tls-certificate', query))
    assert.equal(tlsCertificates[0].get('id'), '123456')
  })
})
```

However, in [`mock-provider/mirage.js`](https://github.com/salsify/ember-cli-pact/blob/master/addon-test-support/mock-provider/mirage.js#L101), ember-cli-pact is looking for `this.schema.modelFor(resource.modelName)`. Unfortunately, the `resource.modelName` value is dasherized here (i.e. `tls-certificate`), and all of the models in the schema are camelcased (i.e. `tlsCertificate`).

Using camelized model names in the test does not have any affect, and all production code uses `this.store.query('tls-certificate')` just fine. Does anyone have any thoughts as to how I can get ember-cli-pact to cooperate with my camelized models or how to change all my models within mirage to be dasherized if that's the only way to go?

**Comment by jamesarosen** [comment]
Interestingly, mirage's `schema` has a [private `_modelFor` that does the camelizing](https://github.com/samselikoff/ember-cli-mirage/blob/2eabc2a285e774d0f1d5d05faef514488639e4fd/addon/orm/schema.js#L325-L332):
```js
/**
  * @method _modelFor
  * @param modelName
  * @private
  */
_modelFor(modelName) {
  return this._registry[camelize(modelName)].class;
}
```

**Comment by dfreeman** [comment]
This should be fixed in the `1.0.0-alpha.5` release that just went out. Thanks @psbanka!
